packages/core/lib/debug/interpreter.js

```diff
@@ -761,6 +778,21 @@ class DebugInterpreter {
     }
     return tempPrintouts;
   }
+
+  // update the numbers of lines to be printed from options -<num>|+<num> from user args
+  updatePrintoutLines(userArgs, currentLines) {
+    let beforeLines = currentLines[0], afterLines = currentLines[1];
+    for (let argument of userArgs) {
+      let newLines = Number(argument.slice(1));
+      if (isNaN(newLines)) continue;
+      if (argument[0] === "-") {
+        beforeLines = newLines;
+      } else if (argument[0] === "+") {
+        afterLines = newLines;
+      }
```

Comment:
So, any arguments that aren't of the correct form will just be ignored.  I think that's a reasonable decision for the debugger (it's not something I'd suggest in most contexts, but in the debugger I think it's OK), but I think it's worth making explicit with a comment.